snuba/query/allocation_policies/__init__.py

```diff
@@ -170,16 +170,25 @@ def quota_allowance(self) -> dict[str, dict[str, Any]]:
             dict[str, dict[str, Any]], self.extra_data.get("quota_allowances", {})
         )
 
+    @property
+    def summary(self) -> dict[str, Any]:
+        return cast(dict[str, Any], self.extra_data.get("summary"))
```

Comment:
Is there a case where the `summary` key won't be in the `extra_data`? If so, I don't think this typing holds true. In that case the function will return `None`.

Reply:
No, there will always be a summary

Reply:
I added an assertion